src/main/kotlin/no/nav/klage/oppgave/service/MottakService.kt

```diff
@@ -116,4 +115,34 @@ class MottakService(
             throw OversendtKlageNotValidException("$journalpostId er ikke en gyldig journalpost referanse")
         }
 
+    private fun validateKlager(klager: OversendtPartId) {
```

Comment:
Hvis vi ikke bruker et lib for dette, så kunne vi kanskje puttet det i en Util-klasse/metode og hatt et par unit-tester?